docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+* **Show pause Contact preferences** - Select **Yes** to enable Contacts to turn-off messages from your Mautic account to their Email address for a specified date range. This action isn’t a full unsubscribe, and at the end of the date range, the message is sent to that address once again.
+
+* **Show Contact’s Categories** - If you have Categories set for Contacts, Campaigns, Emails, etc., select Yes to enable the Contact to opt out of the Categories they choose from the preference center page.
+
+* **Show Contact’s preferred Channel option** - If you have multiple Channels available within your Mautic account (Email, SMS, mobile push, web notifications, etc.), Contacts can choose their preferred Channel. This can be useful if you are using the Marketing Messages feature of Mautic. More information about the Preference Center is available :doc:`here</contacts/preference_center>`.
+
+
 Form settings
 *************
 
+.. image:: images/form-settings.png
+  :width: 600
+  :alt: Screenshot showing Form Settings Configuration in Mautic
+
+* **Do not accept submission from these domain names** - To block Contacts with specific Email domains from submitting your Forms, enter those domains in the dialog box. Select an option on each Form you want to apply this block to. You can restrict either specific Email aliases that belong to a domain or an entire domain. To block the entire domain, you can use wildcards (*).
+
 Contact settings
 ****************
 
 Contact merge settings
 ======================
 
+.. image:: images/contact-merge-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact Merge Settings Configuration in Mautic
+
+* **Merge by unique fields with operator** - If you use more than one unique fields identifiers then you can define the way of the find to merge it. 
+
 Contact list settings
 =====================
 
+.. image:: images/contact-list-settings.png
+  :width: 600
+  :alt: Screenshot showing Contact List Settings Configuration in Mautic
+
+* **Columns** -
+
 Import settings
 ===============
 
+.. image:: images/import-settings.png
+  :width: 600
+  :alt: Screenshot showing Import Settings Configuration in Mautic
+
+* **Automatically import in the background if the CSV has more rows than defined** - The browser/background import determines the number of rows in the CSV if this option is greater than 0.
+
 Segment settings
 ****************
 
+.. image:: images/segment-settings.png
+  :width: 600
+  :alt: Screenshot showing Segment Settings Configuration in Mautic
+
+* **Show warning if Segment hasn’t been rebuilt for X hours** - For all Contacts, dynamic Segments get rebuilt nightly. If there is an error that prevents a Segment from rebuilding, Mautic displays a warning message. This field allows you to configure the allowable length of time between rebuilds, post which the warning message appears.
```

Comment:
```suggestion
* **Show warning if Segment hasn’t been rebuilt for X hours** - Segments are rebuilt when the cron jobs are fired. If there is an error that prevents a Segment from rebuilding, Mautic displays a warning message. This field allows you to configure the allowable length of time between rebuilds, after which the warning message appears.
```

This was Acquia-specific documentation. Please link to the docs about the cron job for segment rebuilding.